service/gcs/core/gcs/storage.go

```diff
@@ -437,17 +424,17 @@ func (c *gcsCore) unmountLayers(id string) error {
 // container with the given ID.
 // These files include directories used for mountpoints in the union filesystem
 // and config files.
-func (c *gcsCore) destroyContainerStorage(id string) error {
-	if err := c.OS.RemoveAll(c.getContainerStoragePath(id)); err != nil {
+func (c *gcsCore) destroyContainerStorage(index uint8, id string) error {
```

Comment:
nit: Since `id` is only used in the error string, can't we get the ID from the index in the error case and not have to pass it in? Same for all other occurrences of this.

Reply:
Done.